test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -394,6 +394,20 @@
         (doseq [f files] (is (= "testable string" (slurp f))
                              "File was replaced"))))))
 
+(deftest initialize!-test-with-keylength-in-settings
+  (let [settings (assoc (testutils/ca-settings (ks/temp-dir)) :keylength 768)]
```

Comment:
We have a `ca-sandbox!` function that combines the `(ca-settings (temp-dir ...))` - might use that instead.